README.md

```diff
@@ -1,43 +1,116 @@
-**COC API Server**
+# Clock of Change - API Server (Backend)
 
-the new COC API is written in nodejs on top of express. 
-All routes reside in /core/restapi.js
+
+This repository contains the API Server - or in other terms Backend - of the Clock of Change.
+
+To find out more about the Clock of Change and Human Connection - the network behind it - visit https://human-connection.org/en/.
+
+## Tech Stack
+
+* Node.js: The API Server is running on [Node.js](https://nodejs.org/en/) - a JavaScript runtime environment for server-side scripting
+* Express.js: We use the [Express.js](https://expressjs.com/en/) framework - a Node.js framework to help build web applications
+* Nodemon: We use [Nodemon](https://nodemon.io/) for development - a handy replacement wrapper for Node.js that automatically restarts the application on file changes
+* MySQL: We use [MySQL](https://www.mysql.com) as our relational database of choice to store our data 
+
+
+## Project Structure & Components
+
+**PROJECT STRUCTURE / DIRECTORIES**
+
+* server.js: The starting point for this Node.js application which starts the server
+* core/: The core directory contains the most important files of the project like database, mailer, router and main controller
+* core/entryController.js: Is the main controller for all the requests
+* core/restapi.js: All routes can be found here, they will also be listed further down
+* mails/: The mails directory holds the mail templates
+* public/: The public directory is not used for now
+
+
+**DATABASE**
+
+We use MySQL for the COC API as our relational database. 
+Currently all of the database related code can be found in the `core/db.js` file.
+This includes the credentials for the database (host, user, password and db name) and can be changed in this file.
+
+TODO: Add description of tables
+
+**MAILER**
+
+The code related to the mail system can be found in the file `core/mailer.js`.
+For the mailer to work the smtp credentials need to be changed in this file as well.
+Then the mailer will work and use the mail templates from `mails/entry/`
+
+
+## Installation
+
+**PREREQUESITES**
+
+Before starting the installation you need to make sure you have the following tools installed:
```

Comment:
Slightly verbose. I would put this paragraph in one sentence and use link formatting. Example:

"""
Make sure that you have a recent version of [node](https://nodejs.org/en/), [yarn](https://yarnpkg.com/en/), [EmberJS](https://www.emberjs.com/), [ruby](https://www.ruby-lang.org/en/), [Redis](https://redis.io/) and [postgresql](https://www.postgresql.org/) installed before you proceed. E.g. we have the following versions:
```sh
$ node --version
v10.4.1
$ yarn --version
1.7.0
$ ember --version
ember-cli: 3.1.4
node: 10.4.1
os: linux x64
```
"""
source: https://github.com/roschaefer/rundfunk-mitbestimmen/edit/master/README.md